inlang.config.js

```diff
@@ -0,0 +1,47 @@
+// filename: inlang.config.js
+
+export async function defineConfig(env) {
+  // importing a plugin
+  const plugin = await env.$import(
+    "https://cdn.jsdelivr.net/gh/jannesblobel/inlang-plugin-po@1/dist/index.js"
+  );
+
+  // most plugins require additional config, read the plugins documentation
+  // for the required config and correct usage.
+  const pluginConfig = {
+    pathPattern:
+      "./rest_framework_simplejwt/locale/{language}/LC_MESSAGES/django.po",
+    referenceResourcePath: null,
+  };
+
+  return {
+    referenceLanguage: "en",
+    languages: [
```

Comment:
Does inlang also make a PR to change this config file when a translator has a new language to add?